idaes_examples/mod/power_gen/NGFC_flowsheet.py

```diff
@@ -767,219 +768,180 @@ def scale_flowsheet(m):
+            'cathode_heat', 'cathode_recycle', 'cathode_blower',
+            'cathode_exhaust_split', 'cathode_expander', 'cathode_HRSG',
+            'cathode_exhaust_translator', 'combustor_mix', 'combustor',
+            'combustor_expander', 'anode_HRSG'
+        ]
+        built_units += power_island_units
 
     # set scaling for unit constraints
-    for name in list_units:
+    for name in built_units:
         unit = getattr(m.fs, name)
-        # mixer constraints
+        # mixer
         if hasattr(unit, 'material_mixing_equations'):
             for (t, j), c in unit.material_mixing_equations.items():
                 iscale.constraint_scaling_transform(c, 1e-3, overwrite=False)
-        if hasattr(unit, 'enthalpy_mixing_equations'):
-            for t, c in unit.enthalpy_mixing_equations.items():
-                iscale.constraint_scaling_transform(c, 1e-3, overwrite=False)
-        if hasattr(unit, 'minimum_pressure_constraint'):
-            for (t, i), c in unit.minimum_pressure_constraint.items():
-                iscale.constraint_scaling_transform(c, 1e-5, overwrite=False)
-        if hasattr(unit, 'mixture_pressure'):
-            for t, c in unit.mixture_pressure.items():
-                iscale.constraint_scaling_transform(c, 1e-5, overwrite=False)
-
-        # separator constraints
-        if hasattr(unit, 'material_splitting_eqn'):
-            for (t, o, j), c in unit.material_splitting_eqn.items():
-                iscale.constraint_scaling_transform(c, 1e-3, overwrite=False)
-        if hasattr(unit, 'temperature_equality_eqn'):
-            for (t, o), c in unit.temperature_equality_eqn.items():
-                iscale.constraint_scaling_transform(c, 1e-2, overwrite=False)
-        if hasattr(unit, 'pressure_equality_eqn'):
-            for (t, o), c in unit.pressure_equality_eqn.items():
-                iscale.constraint_scaling_transform(c, 1e-5, overwrite=False)
-        if hasattr(unit, 'sum_split_frac'):
-            for t, c in unit.sum_split_frac.items():
-                iscale.constraint_scaling_transform(c, 1, overwrite=False)
-
-        # pressurechanger constraints
-
-        if hasattr(unit, "ratioP_calculation"):
-            for t, c in unit.ratioP_calculation.items():
-                iscale.constraint_scaling_transform(c, 1e-5, overwrite=False)
-
-        if hasattr(unit, "actual_work"):
-            for t, c in unit.actual_work.items():
-                iscale.constraint_scaling_transform(c, 1e-6, overwrite=False)
-
-        if hasattr(unit, "isentropic_pressure"):
-            for t, c in unit.isentropic_pressure.items():
-                iscale.constraint_scaling_transform(c, 1e-5, overwrite=False)
-
-        if hasattr(unit, "isentropic"):
-            for t, c in unit.isentropic.items():
-                iscale.constraint_scaling_transform(c, 1e-1, overwrite=False)
-
+        # pressure changer
         if hasattr(unit, "isentropic_energy_balance"):
```

Comment:
Probably outside the scope of this PR, but are all these 'if' checks required? They make sense for a generic unit model where users have options over what terms might be included, but for a flowsheet example I would have thought most of these would be concretely defined (i.e. you know if it is here or not).